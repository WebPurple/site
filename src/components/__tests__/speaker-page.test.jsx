// @flow
import * as React from 'react'
import { shallow } from 'enzyme'
import SpeakerPage from '../speaker-page'

describe('<SpeakerPage>', () => {
  it('should match snapshot', () =>
    expect(
      shallow(
        <SpeakerPage
          speaker={{
            fields: {
              slug: '',
            },
            id: '',
            avatar: null,
            jobTitle: '',
            organization: '',
            socialNetworks: [],
            talks: [],
          }}
        />,
      ),
    ).toMatchSnapshot())
})
