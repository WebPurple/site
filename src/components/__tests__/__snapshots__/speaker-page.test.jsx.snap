// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpeakerPage> should match snapshot 1`] = `
<Flex
  blacklist={
    Array [
      "css",
      "flexWrap",
      "flexDirection",
      "alignItems",
      "justifyContent",
      "css",
      "width",
      "m",
      "mt",
      "mr",
      "mb",
      "ml",
      "mx",
      "my",
      "p",
      "pt",
      "pr",
      "pb",
      "pl",
      "px",
      "py",
      "fontSize",
      "color",
      "bg",
      "flex",
      "order",
      "alignSelf",
    ]
  }
  flexDirection={
    Array [
      "column",
      "row",
    ]
  }
  is={[Function]}
  m={
    Array [
      "2rem 2rem",
      "4.0rem 8.6rem",
      "4.0rem 10.8rem",
      "4.0rem 12rem",
    ]
  }
>
  <Flex
    alignItems="center"
    blacklist={
      Array [
        "css",
        "flexWrap",
        "flexDirection",
        "alignItems",
        "justifyContent",
        "css",
        "width",
        "m",
        "mt",
        "mr",
        "mb",
        "ml",
        "mx",
        "my",
        "p",
        "pt",
        "pr",
        "pb",
        "pl",
        "px",
        "py",
        "fontSize",
        "color",
        "bg",
        "flex",
        "order",
        "alignSelf",
      ]
    }
    flexDirection="column"
    is={[Function]}
  >
    <Box
      blacklist={
        Array [
          "css",
          "width",
          "m",
          "mt",
          "mr",
          "mb",
          "ml",
          "mx",
          "my",
          "p",
          "pt",
          "pr",
          "pb",
          "pl",
          "px",
          "py",
          "fontSize",
          "color",
          "bg",
          "flex",
          "order",
          "alignSelf",
        ]
      }
      mb="16px"
      style={
        Object {
          "height": 200,
        }
      }
    >
      <Avatar
        avatar={null}
      />
    </Box>
    <SpeakerContacts
      speaker={
        Object {
          "avatar": null,
          "fields": Object {
            "slug": "",
          },
          "id": "",
          "jobTitle": "",
          "organization": "",
          "socialNetworks": Array [],
          "talks": Array [],
        }
      }
    />
  </Flex>
  <Box
    blacklist={
      Array [
        "css",
        "width",
        "m",
        "mt",
        "mr",
        "mb",
        "ml",
        "mx",
        "my",
        "p",
        "pt",
        "pr",
        "pb",
        "pl",
        "px",
        "py",
        "fontSize",
        "color",
        "bg",
        "flex",
        "order",
        "alignSelf",
      ]
    }
    ml={
      Array [
        0,
        "60px",
      ]
    }
  >
    <styled.h1 />
    <Flex
      blacklist={
        Array [
          "css",
          "flexWrap",
          "flexDirection",
          "alignItems",
          "justifyContent",
          "css",
          "width",
          "m",
          "mt",
          "mr",
          "mb",
          "ml",
          "mx",
          "my",
          "p",
          "pt",
          "pr",
          "pb",
          "pl",
          "px",
          "py",
          "fontSize",
          "color",
          "bg",
          "flex",
          "order",
          "alignSelf",
        ]
      }
      flexDirection="column"
      is={[Function]}
      m={0}
      mt="72px"
      p={0}
    />
  </Box>
</Flex>
`;
